deploy/vagrant/conf/alluxio.yml

```diff
@@ -4,13 +4,16 @@
 # {Github | Local | Release}
 Type: Local
 
+# If the version is less than 1.1.0, ie. 1.0.1. This should be false for 1.1.0-SNAPSHOT. {true|false}
+Version_LessThan_1.1.0: false
```

Comment:
I agree that the dependency on the version command output is not ideal, but the advantage is that it removes the need for the extra setting, avoiding the possibility for a mismatch between the setting and what the version actually is. Another way to think of it is to compare the two scenarios, when a problem occurs:

1) in the current approach, a problem arises when the value of the boolean property does not correctly identify whether the Alluxio version to be installed is at least 1.1.0 (which is something that we as developers cannot control). When this happens, the error that the user sees will not happen until alluxio-env.sh is being executed.

2) In the approach I propose, a problem arises when the output format of the version command changes (which is something that we as developers can control). The deploy script can fail with an informative message if it detects an output format change.